free-2.md

```diff
@@ -11,34 +11,41 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Many college students struggle to organize multiple homework assignments, classtimes, meetings, big projects with smaller parts, reoccuring due dates, a calendar for events, and a list of all the little things they need to do "at some point" during the week. They use checklists, calendars, todo lists, sticky notes and other resources, but there doesn't seem to be one resources that can take care of all of those needs in one place. I would like to build a DSL that would allow them to merge all of these things they need to do into one calendar, this would decrease stress and help them to more efficiently complete everything that they needed to do.
 
```

Comment:
Do you imagine this being a tool optimized for Mudd students? Or is there a subset of majors/schools that you want to focus on? I think the course load/type of the student could affect your design choices.